src/services/Subscriptions.php

```diff
@@ -87,10 +87,13 @@ public function syncAllSubscriptions(): void
      * This takes the stripe subscription data from the API and creates or updates a Subscription element.
      *
      * @param StripeSubscription $subscription
+     * @param bool $checkForUnsavedDraft
      * @return bool Whether the synchronization succeeded.
      */
-    public function createOrUpdateSubscription(StripeSubscription $subscription): bool
+    public function createOrUpdateSubscription(StripeSubscription $subscription, bool $checkForUnsavedDraft = false): bool
```

Comment:
Probably an edge case that anyone would be extending this class, but if they are and are re-implementing this method they would need to make sure their code is updated as the signature has changed.